sdk/digitaltwins/azure-digitaltwins-core/azure/digitaltwins/core/__init__.py

```diff
@@ -0,0 +1,25 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from ._digitaltwins_client import DigitalTwinsClient
+from ._generated.models._models import DigitalTwinsModelData
+from ._generated.models._models import QueryResult
+from ._generated.models._models import IncomingRelationship
+from ._generated.models._models import EventRoute
```

Comment:
These imports should just be:
`from ._generated.models import QueryResult`

If you want you can open a github issue to fix this in a later release rather than do it now, as it's not super important.

Reply:
Fixed